lib/repl.js

```diff
@@ -330,11 +339,16 @@ function REPLServer(prompt, stream, eval_, useGlobal, ignoreUndefined) {
 inherits(REPLServer, rl.Interface);
 exports.REPLServer = REPLServer;
 
+exports.REPL_MODE_DEFAULT = 0;
```

Comment:
Called non-strict mode "DEFAULT" here, because "SLOPPY" felt somewhat pejorative. Used "sloppy" in the NODE_REPL_MODE env setting, though – is there a better term for this, @domenic?